server/views/macros/inactiveLocationBanner.njk

```diff
@@ -10,6 +10,12 @@
   {% if location.deactivationReasonDescription %}
     {% set deactivatedReason = deactivatedReason + " - " + location.deactivationReasonDescription %}
   {% endif %}
+  {% set changeRef = '#' %}
```

Comment:
I'm removing the links if they cannot manage deactivation - probably a better way with a boolean or something!